packages/squiggle-lang/src/fr/common.ts

```diff
@@ -85,4 +86,30 @@ myFn = typeOf({|e| e})`,
       ),
     ],
   }),
+  maker.make({
+    name: "try",
+    description:
+      "Try to run a function and return its result. If the function throws an error, return the result of the fallback function instead.",
+    definitions: [
+      makeDefinition(
+        [
+          frNamed("fn", frLambdaTyped([], frAny({ genericName: "A" }))),
+          // in the future, this function could be called with the error message
+          frNamed("fallbackFn", frLambdaTyped([], frAny({ genericName: "A" }))),
+        ],
+        frAny({ genericName: "A" }),
+        ([fn, fallbackFn], reducer) => {
+          try {
+            return reducer.call(fn, []);
+          } catch (e) {
+            if (!(e instanceof ErrorMessage)) {
+              // This doesn't looks like an error in user code, treat it as fatal
+              throw e;
+            }
+            return reducer.call(fallbackFn, []);
```

Comment:
I think we'll want to catch the error later, but we don't need that now. Maybe we make a new formal error type.